doc/commentary.txt

```diff
@@ -6,6 +6,9 @@ License: Same terms as Vim itself (see |license|)
 Comment stuff out.  Then uncomment it later.  Relies on 'commentstring' to be
 correctly set, or uses b:commentary_format if it is set.
 
+Assign b:commentary_startofline to insert comment characters into
+beginning of line.
```

Comment:
"Into beginning of line" is really awkward grammar, and I suspect a lot of people reading it wouldn't understand what it means.

Reply:
Assign b:commentary_startofline to insert comment characters at column 1 regardless of indentation:
```
def calc_stuff(val):
    out = val*val
#    print(out)
    return out
```

Would this be ok?

Reply:
The initial sentence is good, ditch the rest.

Reply:
Done